src/FriendlyErrors.php

```diff
@@ -95,5 +96,32 @@ public static function checkTesseractVersion($expected, $action, $command)
 		$msg = join(PHP_EOL, $msg);
 
 		throw new FeatureNotAvailableException($msg);
-	}
+    }
+    
+    public static function checkWritePermissions($path)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The variable $writable_directory is not named in camelCase.](https://app.codacy.com/app/thiagoalessio/tesseract-ocr-for-php/pullRequest?prid=4607615)

Reply:
Your right!
Also `$writable_file`.
Just fixed it.
But the Test failed anyways.

The Test tries to write in `"{$this->imagesDir}/output.hocr"` i am not sure if this works.